css/main.css

```diff
@@ -248,75 +360,113 @@ button {
 }
 
 .obligatory {
-	display: block;
 	float: right;
 	text-align: center;
-	padding-top: 20px;
-	width: 250px;
-	height: 50px;
-	margin-top: 60px;
 	box-sizing: border-box;
 	text-transform: uppercase;
-
+	position: relative;
+	top: 4em;
+	right: 10%;
 }
 
 .visit p,
 .academy p {
 	text-transform: uppercase;
 	text-align: center;
+	position: relative;
+	top: 3em;
 }
 
 .visit,
 .links,
 .academy {
-	display: block;
-	min-height: 90px;
+	height: 8em;
 	width: 400px;
 	box-sizing: border-box;
 	float: left;
 	background-color: #ffffff;
-	padding-top: 30px;
-
 }
 
 .academy p::after {
 	content: "";
 	background-image: url(../img/htmlacademy.png);
-	width: 35px;
-	height: 35px;
+	width: 2em;
+	height: 2em;
 	background-color: #ffffff;
-	display: inline-block;
+	display: block;
 	background-repeat: no-repeat;
 	vertical-align: middle;
+	background-position: center;
+	position: relative;
+	left: 17em;
+	bottom: 2em;
+	box-sizing: border-box;
+}
 
+.academy p:hover::after {
+	background-image: url('../img/academy-hover.png');
 }
 
 .twitter,
 .facebook,
 .youtube {
-
-	display: inline-block;
+	position: relative;
 	background-color: #90b2d0;
-	width: 50px;
-	height: 50px;
-
+	width: 2em;
+	height: 2em;
 }
 
 .twitter {
-	margin-left: 110px;
+	top: 3.5em;
+	left: 7em;
 	background-image: url("../img/icon-twitter.png");
 	background-repeat: no-repeat;
```

Comment:
please move the same properties to a separate class to avoid copy-paste:
_background-repeat: no-repeat;
background-position: center;
background-color: #799cbd;
etc_